src/containerapp/azext_containerapp/_params.py

```diff
@@ -368,6 +368,7 @@ def load_arguments(self, _):
         c.argument('configuration', nargs="*", help="Java component configuration. Configuration must be in format \"<propertyName>=<value>\" \"<propertyName>=<value>\"...")
         c.argument('min_replicas', type=int, help="Minimum number of replicas to run for the Java component.")
         c.argument('max_replicas', type=int, help="Maximum number of replicas to run for the Java component.")
+        c.argument('route_yaml', options_list=['--route-yaml', '--yaml'], help="Path to a .yaml file with the configuration of a Spring Cloud Gateway route. For an example, see https://aka.ms/gateway-for-spring-routes-yaml")
```

Comment:
I tried https://aka.ms/gateway-for-spring-routes-yaml 
But seems it not work.

Reply:
The link will point to our SCG doc, we will release the doc after the cli released, and make the link available.